### PR TITLE
feat: more website components

### PR DESCRIPTION
**Pull request by olegshilov**

### CODE REVIEW

File: libs/website/ui-kit/src/lib/success-message-modal/success-message-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        stroke="#0D0D0E"
+        stroke-opacity="0.24"
+      />
+    </svg>
+  );
+}
+
+export function SuccessMessageModal() {
+  return (
+    <div className="flex items-center justify-between bg-white rounded-[10px] max-w-[343px] sm:max-w-[473px] lg:max-w-[623px] px-[16px] sm:px-[32px] py-[12px] sm:py-[20px] ">
```
Comment (olegshilov) [inline libs/website/ui-kit/src/lib/success-message-modal/success-message-modal.tsx]: remove unnecessary spaces

File: libs/website/ui-kit/src/lib/success-message-modal/success-message-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      <Ruler className="h-full w-auto" />
+      <div className="flex flex-col items-center justify-center ">
+        <Text className="text-center text-haqq-black mb-[24px] sm:mb-[32px] text-[18px] font-medium sm:text-[24px] lg:text-[32px]">
```
Comment (olegshilov) [inline libs/website/ui-kit/src/lib/success-message-modal/success-message-modal.tsx]: use `Heading` component here and wrap it with div with centered text and margins classes

File: libs/website/ui-kit/src/lib/success-message-modal/success-message-modal.tsx (hunk 1 above)
Comment (hadzhehsen) [reply libs/website/ui-kit/src/lib/success-message-modal/success-message-modal.tsx]: Done

File: libs/website/ui-kit/src/lib/success-message-modal/success-message-modal.tsx (hunk 2 above)
Comment (hadzhehsen) [reply libs/website/ui-kit/src/lib/success-message-modal/success-message-modal.tsx]: Done

File: libs/website/ui-kit/src/lib/success-message-modal/success-message-modal.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    </svg>
+  );
+}
+
+export function SuccessMessageModal() {
+  return (
+    <div className="flex items-center justify-between bg-white rounded-[10px] max-w-[343px] sm:max-w-[473px] lg:max-w-[623px] px-[16px] sm:px-[32px] py-[12px] sm:py-[20px]">
+      <Ruler className="h-full w-auto" />
+      <div className="flex flex-col items-center justify-center">
+        <div className="text-center text-haqq-black mb-[24px] sm:mb-[32px] text-[18px] font-medium sm:text-[24px] lg:text-[32px]">
```
Comment (olegshilov) [inline libs/website/ui-kit/src/lib/success-message-modal/success-message-modal.tsx]: only text centering and margins must be here.
```suggestion
        <div className="text-center mb-[24px] sm:mb-[32px]">
```

File: libs/website/ui-kit/src/lib/scroll-button/scroll-button.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+  return (
+    <div className={classNames} onClick={onClick}>
+      <svg
+        width="30"
+        height="10"
+        viewBox="0 0 30 10"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        className="h-"
```
Comment (olegshilov) [inline libs/website/ui-kit/src/lib/scroll-button/scroll-button.tsx]: remove

File: libs/website/ui-kit/src/lib/pagination-button/pagination-button.tsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+export function PaginationButton({
+  className,
+  onClick,
+  active,
+  page,
+}: PaginationButtonProps) {
+  const classNames = clsx(
+    ' text-center align-text-bottom w-[140px] h-[30px] pt-[8px] pb-[0.75px] tracking-tighter border-t-[2px] bg-white',
```
Comment (olegshilov) [inline libs/website/ui-kit/src/lib/pagination-button/pagination-button.tsx]: remove leading space from classNames

File: libs/website/ui-kit/src/lib/page-title/page-title.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  className?: string;
+  children: ReactNode;
+}
+
+export function PageTitle({ className, children }: PageTitleProps) {
+  return (
+    <div
+      className={clsx(
+        styles['container'],
+        className,
```
Comment (olegshilov) [inline libs/website/ui-kit/src/lib/page-title/page-title.tsx]: className from props must be last

Comment (hadzhehsen) [reply libs/website/ui-kit/src/lib/page-title/page-title.tsx]: Done

File: libs/website/ui-kit/src/lib/pagination-button/pagination-button.tsx (hunk 5 above)
Comment (hadzhehsen) [reply libs/website/ui-kit/src/lib/pagination-button/pagination-button.tsx]: Done

File: libs/website/ui-kit/src/lib/scroll-button/scroll-button.tsx (hunk 4 above)
Comment (hadzhehsen) [reply libs/website/ui-kit/src/lib/scroll-button/scroll-button.tsx]: Done

File: libs/website/ui-kit/src/lib/success-message-modal/success-message-modal.tsx (hunk 3 above)
Comment (hadzhehsen) [reply libs/website/ui-kit/src/lib/success-message-modal/success-message-modal.tsx]: Done

**Pull request merged**